CompuCell3D/core/Demos/BookChapterDemos_ComputationalMethodsInCellBiology/Angiogenesis/README.txt

```diff
@@ -0,0 +1,9 @@
+Angiogenesis demonstrates the formation of blood vessels. There are many ways to achieve angiogenesis in CC3D, but this one is a Python-only simulation with only one cell type. It leverages secretion, chemotaxis, and cell-cell adhesion.
```

Comment:
I am not sure if this toy angiogenesis model is accurate. The cells look more sparse than expected.

Reply:
This is also a 3D model and needs to be shown in 3D to make sense.
Might change the comments in the file to make it clear to the user that the model is 3D and should be displayed that way.
Might change the _settings.sqlite file so that the cell field displays as 3D and also include a window showing the diffusion field (VEGF).
![Untitled](https://github.com/CompuCell3D/CompuCell3D/assets/22944062/b1f6ba65-bb44-4732-ab4c-66336deb1373)